dev/DevBuildController.php

```diff
@@ -16,16 +16,15 @@ public function build($request) {
 			$da = DatabaseAdmin::create();
 			return $da->handleRequest($request, $this->model);
 		} else {
-			$renderer = DebugView::create();
-			$renderer->writeHeader();
-			$renderer->writeInfo("Environment Builder", Director::absoluteBaseURL());
-			echo "<div class=\"build\">";
+			if(Director::isDev() || Permission::check("ADMIN")) {
+				$renderer = DebugView::create();
+				$renderer->writeHeader();
+				$renderer->writeInfo("Environment Builder", Director::absoluteBaseURL());
+				echo "<div class=\"build\">";
+			}
 
 			$da = DatabaseAdmin::create();
 			return $da->handleRequest($request, $this->model);
-
-			echo "</div>";
-			$renderer->writeFooter();
```

Comment:
writeFooter() function has never been used use, so I removed it. I can try add this with correct displaying.